Cannot find module 'nx/src/generators/utils/json'

Hi,

Have an NX project 13.10.2 with an angular app running storybook

Followed the setup at: https://www.npmjs.com/package/@twittwer/compodoc#generator

Seeing the following error `Cannot find module 'nx/src/generators/utils/json'` after running `npx nx g @twittwer/compodoc:config`

See screenshot

<img width="1578" alt="Screenshot 2022-08-31 at 21 47 17" src="https://user-images.githubusercontent.com/5794048/187780248-028140d4-e5dc-46bd-8360-f6acdbac2609.png">

The same error appears no matter what I enter as a project name.

Any help, much appreciated.

--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "@nrwl/devkit": "14.8.3",
+    "nx": "14.8.3",
     "tslib": "2.4.0"
   },
   "devDependencies": {
@@ -41,7 +42,6 @@
     "eslint-config-prettier": "8.5.0",
     "husky": "8.0.1",
     "jest": "29.1.2",
-    "nx": "14.8.3",
     "prettier": "2.7.1",
     "ts-jest": "29.0.3",
     "typescript": "4.8.4",

--- a/packages/compodoc/package.json
+++ b/packages/compodoc/package.json
@@ -6,6 +6,7 @@
   "executors": "./executors.json",
   "dependencies": {
     "@nrwl/devkit": "^14.8.3",
+    "nx": "^14.8.3",
     "tslib": "^2.4.0"
   },
   "peerDependencies": {

--- a/scripts/create-test-workspace.sh
+++ b/scripts/create-test-workspace.sh
@@ -1,0 +1,34 @@
+cd ..
+
+rm -rf compodoc-test
+
+NX_VERSION=latest
+# NX_VERSION=13.10.2
+
+npx create-nx-workspace@$NX_VERSION \
+  --name=compodoc-test \
+  --cli=nx \
+  --nxCloud=false \
+  --preset=angular \
+  --appName=app \
+  --style=scss
+
+cd compodoc-test
+
+npx nx generate @nrwl/angular:library \
+  --name=lib
+
+npx nx format
+
+git add .
+git commit -am "Workspace Setup"
+
+npm i -D @twittwer/compodoc@latest
+
+git commit -am "Install @twittwer/compodoc"
+
+npx nx generate @twittwer/compodoc:config \
+  --project=lib
+
+git add .
+git commit -am "Configure compodoc for lib"

